### PR TITLE
Webapp manifest

### LINKED ISSUE #13: Manifest for webapp

### PR DESCRIPTION
**Pull request by comzeradd**
Fixes #13

### CODE REVIEW

**Comment by johngian** [comment]
We need to change the icons to those used in `master`.

File: marketpulse/base/static/manifest.webapp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Mozilla Marketpulse",
+    "description": "Web app to help gather local market data",
```
Comment (johngian) [inline marketpulse/base/static/manifest.webapp]: We should be more descriptive and add that we gather local market data for fxos.

File: marketpulse/main/views.py [hunk 2]
```diff
@@ -78,3 +79,13 @@ def activities(request):
                   {'user': request.user,
                    'total_contributions': total_contributions,
                    'percent': percent})
+
+
+def manifest(request):
+    import os
+    manifest = os.path.join(settings.BASE_DIR, 'static/manifest.webapp')
```
Comment (johngian) [inline marketpulse/main/views.py]: You can simply add the manifest.webapp as a template and render it.

Comment (johngian) [inline marketpulse/main/views.py]: This way we are rendering contribute.json [1] on Mozillians.org.

[1] https://github.com/mozilla/mozillians/pull/1143/files

File: marketpulse/settings/base.py [hunk 3]
```diff
@@ -108,6 +108,9 @@
 MEDIA_ROOT = config('MEDIA_ROOT', default=os.path.join(BASE_DIR, 'media'))
 MEDIA_URL = config('MEDIA_URL', '/media/')
 
+WEBAPP_MANIFEST = config('WEBAPP_MANIFEST',
```
Comment (johngian) [inline marketpulse/settings/base.py]: Are we using this somehow?

**Comment by johngian** [comment]
r+wc :rocket:

**Comment by comzeradd** [comment]
PR updated. @johngian could have an extra look?

**Comment by johngian** [comment]
r+

**Pull request merged**